docs/readme.md

```diff
@@ -0,0 +1,3 @@
+# Welcome to FAST-DNA
+
+FAST-DNA is an accelerator for anyone using the distributed NPM packages.
```

Comment:
I feel like this is missing details; should this align with the root readme paragraph?

Reply:
It is.  I'm not creating new content in this PR.  I don't want to duplicate the root readme.md.  Jason will update over time.

Reply:
I would prefer this be at least a little more verbose because right now it doesn't tell me anything. Perhaps:

"FAST-DNA is a monolithic repository with NPM packages that export UI components and utilities to be used as an accelerator for web sites and applications"

Reply:
Per Jason he will add detail here.